multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -146,3 +146,58 @@ def test_bz787736(self, multihost, create_backup):
         execute_cmd(multihost, "pwck -r > /tmp/anuj")
         assert "no matching password file entry" not in \
                execute_cmd(multihost, "cat /tmp/anuj").stdout_text
+
+    def test_bz_667593(self, multihost):
+        """
+        :title: Shadow-Utils: sg works with password
```

Comment:
You are missing the bugzilla link in the description.

Reply:
Fixed